packages/launchpad/cypress/e2e/migration.cy.ts

```diff
@@ -80,6 +80,27 @@ function renameSupport (lang: 'js' | 'ts' | 'coffee' = 'js') {
   }, { lang })
 }
 
+function stubVideoHtml (): void {
+  // ctx.migration.getVideoEmbedHtml
+  cy.withCtx((ctx, o) => {
+    o.sinon.stub(ctx.migration, 'getVideoEmbedHtml').callsFake(async () => {
+      return '<span>Stubbed Video Content</span>'
+    })
+  })
+}
+
+function unstubVideoHtml (): void {
+  cy.withCtx((ctx, o) => {
+    const restoreFn = (ctx.migration.getVideoEmbedHtml as SinonStub).restore
+
+    restoreFn && restoreFn()
+  })
+}
+
+beforeEach(() => {
+  stubVideoHtml()
```

Comment:
Are there any other specs where we we end up loading this video from vimeo for? Good stuff, just want to make sure we're using it everywhere we should. :+1: